### PR TITLE
Fix cross repo pull requests

### LINKED ISSUE #79: Pulls: pull repo info from pull.base, not pull.head
A pull request is a thing on "base" repo, not the "head" repo. In a cross-repo pull, the "base" repo is the one that matters.

Change our code to use the info from the "base" repo.

### PR DESCRIPTION
**Pull request by MooreGuy**
When creating a cross-repo pull request, the repo of the pull should be derived from the base branch, not the head branch.

Otherwise we will get 404's from github since we'll be looking for things that don't exist in the wrong repos.

Closes #79

### CODE REVIEW

**Comment by danielbeardsley** [comment]
I linked the corresponding issue.

File: lib/refresh.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ issueQueue.pop(function(githubIssue, next) {
    gitManager.parseIssue(githubIssue)
    .then(dbManager.updateAllIssueData)
    .done(function() {
-      debug("done refreshing issue %s", githubIssue.number);
+      debug("done refreshing issue %s in repo %s", githubIssue.number, githubIssue.repo);
```
Comment (danielbeardsley) [inline lib/refresh.js]: Does `githubIssue` have a `repo` property?

**Comment by danielbeardsley** [comment]
CR 👍 

But deploy_block 👍  on a question.

File: lib/authentication.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ module.exports = {
                req.logIn(user, function(err) {
                   debug('Got login from ' + user.username);
 
-                  // Redirect user to original URL. Fallback to '/' if DNE.
+                  // Redirect user to original URL. Fallback to '/' if DONE.
```
Comment (copperwall) [inline lib/authentication.js]: This was used as "Does Not Exist", not "DONE"

**Comment by soriat** [comment]
CR ⚡️ 
_btw_: PullDasher linked this as https://github.com/MooreGuy/pulldasher/issues/111

**Comment by copperwall** [comment]
CR :fire:

**Comment by soriat** [comment]
CR ⚡️

File: lib/authentication.js (hunk 2 above)
Comment (MooreGuy) [reply lib/authentication.js]: ohhh

File: lib/refresh.js (hunk 1 above)
Comment (MooreGuy) [reply lib/refresh.js]: I'd hope so, git manager calls parse issue above which accesses it.

**Comment by MooreGuy** [comment]
QA :+1: 
Everything seems to start just fine.

**Pull request merged**